core/capabilities/fakes/http_action.go

```diff
@@ -135,28 +135,20 @@ func (fh *DirectHTTPAction) SendRequest(ctx context.Context, metadata commonCap.
 	return response, nil
 }
 
-func (fh *DirectHTTPAction) Start(ctx context.Context) error {
+func (fh *DirectHTTPAction) start(ctx context.Context) error {
 	fh.eng.Infow("Http Action Start Started")
 	return nil
 }
 
-func (fh *DirectHTTPAction) Close() error {
+func (fh *DirectHTTPAction) close() error {
 	fh.eng.Infow("Http Action Close Started")
 	return nil
 }
 
-func (fh *DirectHTTPAction) Name() string {
-	return HTTPActionServiceName
-}
-
 func (fh *DirectHTTPAction) Description() string {
 	return directHTTPActionInfo.Description
 }
 
-func (fh *DirectHTTPAction) Ready() error {
```

Comment:
@akhilchainani I had to change some of this so that it uses the default behavior for a test. Assume that overwriting it wasn't intentional.